components/higher-order/with-filters/README.md

```diff
@@ -0,0 +1,27 @@
+withFilters
+==============
+
+`withFilters` is a part of [Native Gutenberg Extensibility](https://github.com/WordPress/gutenberg/issues/3330). It is also a React [higher-order component](https://facebook.github.io/react/docs/higher-order-components.html).
+
+Wrapping a component with `withFilters` provides a unique `instanceId` to serve this purpose.
```

Comment:
Is the instance ID bit relevant to someone using this higher-order component? Is it even true (looking at the implementation)? Maybe drop this sentence.

Reply:
Nope, I copied it and forgot to update ...

Reply:
Fixed.